include/rocksdb/c.h

```diff
@@ -936,6 +940,13 @@ extern ROCKSDB_LIBRARY_API rocksdb_iterator_t* rocksdb_writebatch_wi_create_iter
     rocksdb_iterator_t* base_iterator,
     rocksdb_column_family_handle_t* cf);
 
+/* Options utils */
+extern ROCKSDB_LIBRARY_API void load_latest_options(
```

Comment:
I don't know why, but it looks like everything else has "rocksdb_" prefix, so can we call it `rocksdb_load_latest_options()`?